CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-typography:** Note in usage docs about unused italic and demi fonts.
```

Comment:
I think you hated this so much you forgot the [PATCH]  🤣

Reply:
Sonofa… I'm going to fix it and direct push.